docs/overview/mana.md

```diff
@@ -0,0 +1,48 @@
+
+Lastly, using fees is error prone. Because the fee is paid for in the blockchain's native token, it can used for the fee and be transfered at the same time. There have been several instances where large fees were accidentally paid for small transactions. One person once spent [$3.1 million](https://www.coindesk.com/business/2023/11/23/bitcoin-sender-struck-with-31m-transaction-fee-largest-in-history/) on a Bitcoin transaction fee. Because Mana is not KOIN, you cannot make such mistakes on Koinos. Your KOIN is always safe!
+
+---
+## What is Mana?
+Now that we know why fees are a problem and how Mana fixes those problems, we can talk about what Mana is exactly.
+
+Mana is a property of a KOIN token. Every single KOIN can have an equal amount of Mana. If you have 1 KOIN, you also have up to 1 Mana. Likewise, if you have 10 KOIN, you also have up to 10 Mana. Sometimes it is helpful to think of Mana as an absolute value (10 Mana) and other times it is helpful to think of it as a percent (100%). If you have 10 KOIN and your Mana is at 50%, you have 5 Mana and vis versa.
+
+---
+## How do I use Mana?
+Mana is automatically used when you use Koinos. Depending on what your transaction does, it uses different amounts of Mana. In general, the more your transaction does, the more Mana it will use. For example, if you have 10 KOIN with 100% Mana (10 Mana) and you send a transaction that uses 2 Mana, you will have 8 Mana left (80%). But you still have 10 KOIN. This is where we get 80% because 8 Mana divided by 10 KOIN is 80%.
+
+---
+## How do I get Mana?
+As we have shown previously, every KOIN has Mana attached to it and when you use Koinos, some of that Mana is used. However, you also regenerate Mana passively over time. Specifically, you regenerate 20% of your total Mana every day until it is fully recharged. By regenerating 20% of your Mana per day, at most, it will take 5 days to regenerate back to 100% Mana.
+
+In our previous example, you sent a transaction that used 2 Mana, leaving you with 8 Mana (80%). Every day you will regenerate 20% of your Mana. In this case, after 6 hours, you will regenerate 5%, putting you at 85% (8.5 Mana). This will continue until you are back at 100% (after 24 hours).
+
+Because Mana regeneration is 20% per day, it is always relative to the total amount of KOIN you have. If you have 1 KOIN you will renerate 0.2 Mana per day (20% of 1 KOIN). If you have 1,000 KOIN you still regenerate 20% of your Mana per day, but now it is 200 Mana per day.
+
+---
+## What happens if I run out of Mana?
+In short, if you run out of Mana, you can no longer use Koinos. But remember, that is only temporary because your Mana is always regenerating. And if you don't have the time, or don't want to wait, you can always acquire more KOIN to increase your Mana regeneration rate to allow to use Koinos sooner.
```

Comment:
It would always be nice to let the user know that acquiring KOIN comes with 100% mana upon arrival.